lib/utils/Utils.js

```diff
@@ -69,6 +69,25 @@ var Utils = {
             }
         }
         return currentClosestElement;
+    },
+
+    splitOnMissingValues: function(data, keys) {
+        const segments = [];
```

Comment:
> The const declaration creates a read-only reference to a value. It does not mean the value it holds is immutable, just that the variable identifier cannot be reassigned. For instance, in case the content is an object, this means the object itself can still be altered.
> \- [MDN's const documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#Description)

[Object.freeze](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze) prevents modifying an object.